src/coloris.js

```diff
@@ -502,7 +502,6 @@
 
     chroma = (chroma + m);
     x = (x + m);
-    m = m;
```

Comment:
Hahaha, that's what happens when I blindly follow the algorithms I found on wikipedia!!